rustls/src/tls13/key_schedule.rs

```diff
@@ -578,26 +578,30 @@ impl KeyScheduleTraffic {
             .export_keying_material(&self.current_exporter_secret, out, label, context)
     }
 
-    pub(crate) fn extract_secrets(&self, side: Side) -> Result<PartiallyExtractedSecrets, Error> {
-        fn expand(
-            secret: &OkmBlock,
-            hkdf: &'static dyn Hkdf,
-            aead_key_len: usize,
-        ) -> (AeadKey, Iv) {
-            let expander = hkdf.expander_for_okm(secret);
-
-            (
-                hkdf_expand_label_aead_key(expander.as_ref(), aead_key_len, b"key", &[]),
-                hkdf_expand_label(expander.as_ref(), b"iv", &[]),
-            )
-        }
+    pub(crate) fn refresh_traffic_secret(
```

Comment:
I think this rename means this commit won't compile, can you fix that?

Reply:
This was not a rename (if you look at the diff you'll notice `extract_secrets` still exists below). However, `refresh_traffic_secrets` is part of the wrong commit here, so I've moved the diff adding it to [99b1da4](https://github.com/rustls/rustls/pull/2370/commits/99b1da43204855335856f358668fd2276e526aa4).